### PR TITLE
! Some HTML5 validation fixes

### PR DESCRIPTION
**Pull request by Arantor**
Signed-off-by: Peter Spicer sleeping@myperch.org

### CODE REVIEW

**Comment by Xarcell** [comment]
What's this? Your back so soon?

**Comment by Arantor** [comment]
I didn't say I was back. I have not said I'm back. Technically I'm not. The simple truth is that I'm trying to figure out how many bugs there are (because whether I'm back or whether I'm not, I am going to obsess over it, freak out over it, which is no small part of why I'm where I am... would prefer not to escalate it to a full breakdown), and if it is clear what are bugs, what are not bugs etc. then maybe it can be resolved. I just fixed these few lines while investigating a years-old bug report.

Though the notion that I should have to defend, or justify, my actions to anyone is also inherently problematic. Is it really too much that I should just do things without having to feel like I'm being judged on what I'm doing?

**Comment by Xarcell** [comment]
I was just teasing. It looked though you fixed something here, hence the remark post in this issue none of the others. So I wanted to tease you about it. Geez, just drag me out back and shoot me for it. I think you should take more time off to do things without expectations from people. 

You make me imagine a guy who sits in a sniper tower all day waiting for someone to peek over the fence. <-- another joke here <--

Do I annoy you that much? Don't answer that. I'm merely jesting, trying to lighten the mood, but with you I'm probably making things worse. Although, I have to say it hurts my feelings every time you bite my head off.

Serious for a moment though, you have a chip on your shoulder dude, but I don't blame you for having it. I think too much depends on you and people expect too much from you, but I also think you set high standards for yourself, and people see that. I understand fully what your going through, because I go through it in real life, so I can relate.

Bottom line is I just want you to be happy. Cheers...

**Comment by Arantor** [comment]
No, it's not you personally that annoys me. It's my own nature that screws me up.
